### DISCUSSION TITLE
keyboard navigation of popup

### DISCUSSION THREAD

**Discussion by eleanorkonik** [ebullient/obsidian-task-collector#46] [Q&A] [answered]
I love that I can set a hotkey to pop up the selection modal, but is there any way to navigate it with arrows so I don't have to use the mouse? 👀

**Comment by ebullient** [comment, marked as answer]
just type the letter (shown as a value next to the checkbox):
`[HotKey] >` or `[HotKey] ?`

**Comment by eleanorkonik** [comment]
ohhhhh, thank you!
